charts/argo-cd/Chart.yaml

```diff
@@ -22,3 +22,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Support AWS ALB Ingress with gRPC"
```

Comment:
This should have been removed for this release as it was part of the previous change.

Reply:
ah, I only looked at the readme. Maybe this can be emphasized in the template?

Reply:
No worries :slightly_smiling_face:  . Agree, we should make this clearer for folks pushing changes :+1: